src/clj/game/cards/hardware.clj

```diff
@@ -394,6 +394,18 @@
                 {:msg "remove 1 tag" :label "[Trash]: Remove 1 tag"
                  :effect (effect (trash card {:cause :ability-cost}) (lose :tag 1))}]}
 
+   "Friday Chip"
+    (let [ability {:msg (msg "move 1 virus counter to " (:title target))
+                   :req (req (pos? (get-in card [:counter :virus] 0)))
+                   :choices {:req #(and (has-subtype? % "Virus")
+                                        (pos? (get-in % [:counter :virus] 0)))}
+                   :effect (req (when (pos? (get-virus-counters state side target))
+                                  (add-counter state side card :virus -1)
+                                  (add-counter state side target :virus 1)))}]
+     {:events {:runner-turn-begins ability
+               :runner-trash {:msg "to gain a virus counter"
+                              :effect (effect (add-counter card :virus 1))}}})
```

Comment:
Should we make this optional? There are now reasons to not gain virus counters (Reverse Infection)... though I suppose the runner has plenty of time to manually remove a counter after trashing a corp card, before ending their turn and letting the corp act?